tests/system.py

```diff
@@ -309,15 +309,6 @@ def test_get_set_iam_policy(self):
         self.assertEqual(returned_policy.version, 3)
         self.assertEqual(returned_policy.bindings, policy.bindings)
 
-        with pytest.raises(
-            BadRequest, match="cannot be less than the existing policy version"
-        ):
-            bucket.get_iam_policy()
```

Comment:
Do we have a definite confirmation that the backend no longer raises this exception under any circumstance?

Reply:
We have an internal group with a conversation. Yes.